pkgs/desktops/gnome-3/extensions/pop-os-shell/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, nodePackages, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "pop-os-shell";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev =  "1.0.0";
+    sha256 = "sha256-gv9knhG1/HOUsIyPWfBRBpJWg/bfRXbbg/x2QDgSmls=";
+  };
+
+  buildInputs = [ glib nodePackages.typescript ];
+
+
+  uuid = "pop-shell@system76.com";
```

Comment:
is this an env that is required for something?

Reply:
I'm not sure. Just saw this being set in all the other extensions. 
Should i remove it?

Reply:
I would remove it if it's not required